management/src/test/java/org/ehcache/docs/ManagementTest.java

```diff
@@ -99,8 +99,10 @@ public void usingManagementRegistry() throws Exception {
 
         // hit count is a sampled stat, for example its values could be [0,0,3,4].
         // In the present case, only the last value is important to us , the cache was eventually hit 4 times
-        int mostRecentIndex = onHeapStore_Hit_Count.getValue().length - 1;
-        onHeapHitCount = onHeapStore_Hit_Count.getValue()[mostRecentIndex].getValue();
+        if (onHeapStore_Hit_Count.getValue().length > 0) {
```

Comment:
btw @ljacomet the other error (50 vs 55 for the size of descriptors : @gibsong and I looked into it, and we also took it to @chrisdennis too noticing some weird stats fwk 1.20 removal of stats registration (OnHeap:Eviction stats are sometimes missing , when using a disk or offheap store). No definitive answers yet, we could timebomb the incriminated tests if you want.
I'll also make sure the failure is more explicit (something like "OnHeap:Eviction stats are missing" instead of "got 50 was expecting 55")